api_examples/sequential_learning_api_tutorial.ipynb

```diff
@@ -0,0 +1,692 @@
+    "\n",
+    "In a real example, we could minimize or maximize any output: compressive strengths, conductivities, and so on."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def toy_func(inputs):\n",
+    "    func_sum = 0.0\n",
+    "    for inp in inputs:\n",
+    "        func_sum += inp**2\n",
+    "    return func_sum\n",
+    "\n",
+    "toy_x = [np.random.normal(loc=3.0, scale=1.0, size=(1, 2))[0] for x in range(10)]\n",
+    "toy_y = [toy_func(x) for x in toy_x]\n",
+    "\n",
+    "initial_best_measured_value = min(toy_y)\n",
+    "\n",
+    "print(f\"Best (lowest) value in initial training set: {initial_best_measured_value}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can plot the initial training set, and color it by the function value"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot initial training set, colored by toy function values\n",
+    "\n",
+    "plt.scatter(np.array(toy_x)[:,0], np.array(toy_x)[:,1],\n",
+    "            c=toy_y, cmap=plt.cm.plasma)\n",
+    "plt.xlabel(\"X1\")\n",
+    "plt.ylabel(\"X2\")\n",
+    "plt.xlim(-5,5)\n",
+    "plt.ylim(-5,5)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "# Write a PIF JSON dataset file\n",
+    "write_dataset_from_func(toy_func, \"toy_initial_dataset.json\", toy_x)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Make a dataset, upload to Citrination, return/print the ID\n",
+    "\n",
+    "dataset_id = upload_data_and_get_id(\n",
+    "    client,\n",
+    "    \"toy_initial_dataset\",\n",
+    "    \"toy_initial_dataset.json\",\n",
+    "    create_new_version=True,\n",
+    "#   given_dataset_id=1234\n",
+    ")\n",
+    "print(f\"Dataset created: {dataset_id}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "# Make a data view on Citrination, return/print the ID\n",
+    "# view_id = 1234 # If view exists and you're re-running this notebook\n",
+    "\n",
+    "view_id = build_view_and_get_id(client, dataset_id, \"toy_view\", view_desc=\"toy test view\",\n",
+    "                      input_keys=[\"Property x1\", \"Property x2\"], output_keys=[\"Property y\"])\n",
+    "\n",
+    "print(f\"Data view created: {view_id}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Running design\n",
```

Comment:
In this section, maybe also add a comment+internal link to read comments in `run_design()` so people know what they're actually doing (for those who care), because just this is pretty abstracted.